lib/public/IConfig.php

```diff
@@ -216,6 +216,18 @@ public function getUserValueForUsers($appName, $key, $userIds);
 	 */
 	public function getUserKeys($userId, $appName);
 
+	/**
+	 * Get all user configs sorted by app of one user
+	 *
+	 * @param string $userId the userId of the user that we want to get all values from
+	 * @return array[] - 2 dimensional array with the following structure:
```

Comment:
Psalm return docs to make it easier?

Reply:
```suggestion
	 * @return array<string,array<string,string>> - 2 dimensional array with the following structure:
```
Like this?

Reply:
No, it's additionally:

```suggestion
	 * @psalm-return array<string, array<string, string>>
	 * @return array[] - 2 dimensional array with the following structure:
```